examples/00-load/create-polyhedron.py

```diff
@@ -0,0 +1,196 @@
+    [0.03, 0.0, 0.02],  # 21
+    [0.02, 0.0, 0.03],  # 22
+    [0.02, 0.01, 0.03],  # 23
+    [0.03, 0.01, 0.03],  # 24
+    [0.035, 0.005, 0.03],  # 25
+    [0.03, 0.0, 0.03],  # 26
+]
+points = quad_points + polygon_points + hexa_points + polyhedron_points
+
+
+###############################################################################
+# Cell connectivity
+# ~~~~~~~~~~~~~~~~~
+# Connectivity describes the indices of the points to compose each cell. The
+# first item in each cell's connectivity is the number of item the cell will
+# have. For example, a quad cell is composed of point ``[0, 1, 2, 3]`` and
+# totaling 4 points, therefore ``[4, 0, 1, 2, 3]`` describes its connectivity.
+#
+# .. note::
+#    This example uses lists for simplicity, but internally PyVista converts
+#    these lists to a :class:`numpy.ndarray` with ``dtype=pyvista.ID_TYPE`` and
+#    passes it to VTK.
+#
+# The same approach can be applied to all the other cell types.
+
+quad = [4, 0, 1, 2, 3]
+polygon = [5, 4, 5, 6, 7, 8]
+hexa = [8, 9, 10, 11, 12, 13, 14, 15, 16]
+
+
+###############################################################################
+# Polyhedron connectivity array
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The connectivity array of polyhedra is defined differently from the rest of the cell
+# types. For polyhedra, we need to set the faces with the following format:
+#
+# ``[NItems, NFaces, Face0NPoints, Face0Point0, Face0Point1..., Face0PointN, Face1NPoints, ...]``
+#
+# Where:
+#
+# - ``NItems`` refers to the total number of items in the list needed to
+#   describe the polyhedron.
+# - ``NFaces`` is the number of faces the polyhedron will have.
+# - ``Face1Npoints`` is the number of points the first face will have
+# - ``Face1Point1...Face1PointN`` are each of the points that describe ``face0``
+#
+# In ``polyhedron_connectivity``, the first item is ``NFaces``. ``NItems`` is
+# added to ``polyhedron``.
+
+polyhedron_connectivity = [
+    # NItems will go here
+    7,  # number of faces
+    5,  # number of points in face0
+    17,  # point index 0
+    18,  # point index 1
+    19,  # point index 2
+    20,  # point index 3
+    21,  # point index 4
+    4,  # number of points in face1
+    17,  # point index ...
+    18,
+    23,
+    22,
+    4,
+    17,
+    21,
+    26,
+    22,
+    4,
+    21,
+    26,
+    25,
+    20,
+    4,
+    20,
+    25,
+    24,
+    19,
+    4,
+    19,
+    24,
+    23,
+    18,
+    5,
+    22,
+    23,
+    24,
+    25,
+    26,
+]
+
+# note how we retroactively add NItems
+polyhedron = [len(polyhedron_connectivity)] + polyhedron_connectivity
+
+
+###############################################################################
+# Cells array
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Now we build the input cells array for the
+# :class:`pyvista.UnstructuredGrid`. Here, we join all cells in a one dimensional
+# array. Internally, the ``NElements`` previously described is used to determine
```

Comment:
If we no longer have arrays then I'd prefer to have these called lists everywhere...
```suggestion
# list. Internally, the ``NItems`` previously described is used to determine
```
Then again "one dimensional" would probably have to be rephrased as "flat".

Reply:
> Then again "one dimensional" would probably have to be rephrased as "flat".

Done